app/views/partials/head.ejs

```diff
@@ -1,31 +1,93 @@
-<%
```

Comment:
These variables should not be removed, all the meta-information will be empty.

Reply:
Oops, fixed. Removed them so I could format the rest nice, prettier seems to be annoying with ejs files. More than really helpful.